docs/user-guide/projects/project-status/README.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_navigation:
+  title: Project status
+  priority: 499
+description: How to set and display the project status.
+robots: index, follow
+keywords: project status
+---
+# Project status
+
+The project status serves as an information for yourself and the team if the project is on track to then being able to quickly act in case it is off track.
+
+##Setting the project status
+
+You can set your project status in the **project settings**. For this, click on **project settings** in the menu on the left and then select the relevant status.
+
+![project-settings-project-status](project-settings-project-status.png)
+
```

Comment:
Please add the information that you can add any kind of additional status description manually to the selected status. Such as a description of the changes, project's KPIs, ...